### PR TITLE
Bug fix of call home export data and enabled callhome for debezium

### PR DESCRIPTION
**Pull request by sanyamsinghal**
[DB-6559]: Enabled callhome for debezium data export
[DB-6560]: Bug fix of sending callhome stats in case of export data command

### CODE REVIEW

**Comment by sanyamsinghal** [comment]
Oracle Automation Test Result: https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/444/
Status:

File: yb-voyager/cmd/exportData.go [hunk 1]
```diff
@@ -80,6 +80,12 @@ func exportData() {
 	success := exportDataOffline()
 
 	if success {
+		tableRowCount := datafile.OpenDescriptor(exportDir).TableRowCount
+		printExportedRowCount(tableRowCount)
+		callhome.GetPayload(exportDir)
```
Comment (rahulb-yb) [inline yb-voyager/cmd/exportData.go:85]: Just add a comment here for clarification that it initialises the required in-memory payload structure along with the file/filepath if not already present- this should fix the issue 👍🏽 
All other calls to this function explicitly make use of the Payload struct it retrieves

**Review by rahulb-yb — APPROVED** [review]
LGTM, just one small comment

**Review by amit-yb — APPROVED** [review]
LGTM.

File: yb-voyager/cmd/exportData.go (hunk 1 above)
Comment (sanyamsinghal) [reply yb-voyager/cmd/exportData.go:85]: updated the description of `GetPayload()`function itself

**Pull request merged**